notebooks/data-sources/oc-github-repo/github_PR_EDA.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<p>Is it 0.07? I see 0.11 in the chart</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/287/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>

Reply:
<p>Updated the markdown 👍</p>